build-system/tasks/css.js

```diff
@@ -54,6 +54,11 @@ const cssEntryPoints = [
     // than the JS file to avoid loading CSS as JS
     outCss: 'video-autoplay-out.css',
   },
+  {
+    path: 'amp-story-player.css',
+    outJs: 'amp-story-player.css.js',
+    outCss: 'amp-story-player-styles.css',
```

Comment:
How about `amp-story-player-v0.css`, to match the `amp-story-player-v0.js`?

Reply:
nit: I think it might make sense to include the `v0` suffix, since this is our versioning, in case we ever need to make a breaking change.